.circleci/config.yml

```diff
@@ -90,6 +90,14 @@ jobs:
     environment:
       TARGET: Linux-X86
       PLATFORM: 1
+  Linux-X86-Clang:
+    <<: *build_and_test_defaults
+  Linux-X86-Clang-platform:
+    <<: *build_and_test_defaults
+    environment:
+      TARGET: Linux-X86-Clang
+      RELEASE: 1
```

Comment:
You can remove this line. We don't support `RELEASE=1` on CircleCI. I'm not sure if it's useful, but if it is, I'll add it later on.

Reply:
Ah, alright. Sounds good.